app/scripts/models/user.js

```diff
@@ -524,6 +530,19 @@ define(function (require, exports, module) {
           }
           return exists;
         });
+    },
+
+    beginFlow: function (flowBeginTime) {
+      if (! this.has('flowId')) {
```

Comment:
Would `this.set('flowId',...` need to be called here if the flowId is only tracked by the flow model?